dff/script/core/normalization.py

```diff
@@ -192,13 +187,13 @@ def normalize_keywords(
 @validate_arguments
 def normalize_script(script: Dict[LabelType, Any]) -> Dict[LabelType, Dict[LabelType, Dict[str, Any]]]:
     """
-    This function normalizes `Script`: it returns `dict` where the `GLOBAL` node is moved
-    into the flow with the `GLOBAL` name. The function returns the structure
+    This function normalizes :py:class:`.Script`: it returns dict where the GLOBAL node is moved
```

Comment:
Maybe add `~` so that it is not displayed as `dff.script.Script`?
```
:py:class:`~.Script`
```
If this is fixed, fix in other places too.

Reply:
agree with @pseusys , i have created task in trello https://trello.com/c/YkTYtvkl